### DISCUSSION TITLE
Broken search bar when browsing code

### DISCUSSION THREAD

**Discussion by nuudlman** [community/community#74505] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.137.1 (114555847)
Device: iPad Pro 11-inch 2 (iOS 17.0.3)
</details>

See image:
![image](https://github.com/community/community/assets/62861466/4f49588c-79b5-4ef1-961c-9174410c5b29)

**Comment by eliperkins** [comment]
Hey @nuudlman, thanks for passing this bug along! I'm able to reproduce it myself, and I've created an issue for this in our internal tracking repo. I'll let you know when this is fixed.

Thanks for helping improve GitHub Mobile!

**Reply by carry0987** [reply]
Same issue on my iPad, see the screenshots, both happened on file list and source code page

![image](https://github.com/community/community/assets/12984973/7371400b-98fb-4430-99fa-e49029dbdbff)

![image](https://github.com/community/community/assets/12984973/1f4b3b25-7a83-427b-a00f-70ad8fda02e4)
